src/dlgTriggerEditor.cpp

```diff
@@ -193,6 +193,50 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
 
     mudlet::loadEdbeeTheme(mpHost->mEditorTheme, mpHost->mEditorThemeFile);
 
+    edbee::StringTextAutoCompleteProvider* provider = new edbee::StringTextAutoCompleteProvider();
+    //QListIterator<QString> i(mudlet::);
+    foreach(QString key, mudlet::mLuaFunctionNames.keys())
```

Comment:
`foreach` is an outdated Qt macro - go with `for (auto key : mudlet::mLuaFunctionNames.keys())`